.github/workflows/ci.yaml

```diff
@@ -0,0 +1,12 @@
+name: CI
+
+on:
+  push:
+    branches:
+      - main
+  pull_request:
+  workflow_dispatch:
+
+jobs:
+  ci:
+    uses: nextstrain/.github/.github/workflows/pathogen-repo-ci.yaml@master
```

Comment:
FYI, the phylo CI workflow is still [fetching data from nextstrain.org](https://github.com/nextstrain/yellow-fever/actions/runs/11963123013/job/33396985682#step:10:35). 

If the desire is to use example data, a couple things to fix up 
- add phylogenetic/example_data/sequences.fast + phylogenetic/example_data/metadata.tsv
- add the rule order at https://github.com/nextstrain/yellow-fever/blob/234292dbefd0fd4afc09ee0e67580193c904da4b/phylogenetic/build-configs/ci/copy_example_data.smk#L18
- include custom rules in the phylogenetic/Snakefile

Reply:
done, thanks.